pom.xml

```diff
@@ -51,6 +51,7 @@
     <properties>
         <che.docs.version>6.12.0-SNAPSHOT</che.docs.version>
         <che.lib.version>6.12.0-SNAPSHOT</che.lib.version>
+        <che.ls.jdt.version>0.0.1-SNAPSHOT</che.ls.jdt.version>
```

Comment:
as we agreed with @tsmaeder `che.ls.jdt` must be tagged version as it will have it's own life cycle and separa release cycle.

Reply:
That is not quite correct, it must be a taggged version when we RELEASE che, not for merging.

Reply:
Which release of Che did you mean - 6.12.0? 
Looks like it should be possible to release `che.ls.jdt` with dependency on Che of version 6.11.0 which had already released.

Reply:
@tsmaeder you are right we need tag only when we do release, however please note that if we will not have tag of it before next release, release will be blocked by thigs.

Reply:
0.0.1 tag is already available please remove the snapshot

Reply:
so since we have tag already this is not blocking anymore